<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartCreator</class>
 <widget class="QMainWindow" name="ChartCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Δημιουργία Γραφήματος</string>
  </property>
  <property name="locale">
   <locale language="Greek" country="Greece"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Τύπος Γραφήματος:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chartType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Γραμμή</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Μπάρες</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Στοίβα</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Πίτα</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chartTheme"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="ChartContainer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="labelsContainer">
      <item>
       <widget class="QPushButton" name="allButton">
        <property name="text">
         <string>Όλα</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delimiterButton">
        <property name="text">
         <string>Διαχωρισμός</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="expression">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ConfirmButton">
        <property name="text">
         <string>Αποθήκεση</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>Ακύρωση</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
